### PR TITLE
try alternative pymc objct backtracing

### PR DESCRIPTION
**Pull request by OriolAbril**
do not merge, testing thing out

### CODE REVIEW

**Comment by OriolAbril** [comment]
Here is how this version looks like right now: https://pymc--276.org.readthedocs.build/projects/examples/en/276/object_index.html

**Comment by OriolAbril** [comment]
This should be ready to merge now

File: examples/conf.py [hunk 1]
```diff
@@ -69,6 +69,7 @@
         {"name": "API", "url": "https://docs.pymc.io/en/stable/api.html"},
     ],
     "page_sidebar_items": ["postcard", "page-toc", "edit-this-page"],
+    "google_analytics_id": "G-6KPRBTE6WV",
```
Comment (twiecki) [inline examples/conf.py:72]: Which analytics account does this record to?

Comment (OriolAbril) [reply examples/conf.py:72]: there are 3 "properties" inside the pymc devs google account: versioned docs, pymcon and examples (this one)

**Review by twiecki — APPROVED** [review]

**Pull request merged**